### ISSUE TITLE
Bug Report - 20171103

### ISSUE THREAD

**Issue by vykrum** [DynamoDS/Dynamo#8310]
## Dynamo version
Dynamo Core 2.0.0.3026

## Operating system
Windows 10

## What did you do? 
In a code block, included '_Enter_' between quotes and didn't terminate the line with a semi colon
![20171103-2](https://user-images.githubusercontent.com/5621624/32378480-dca50044-c0d0-11e7-9849-32a448a30195.png)

## What did you expect to see?
The code block to automatically include the semi colon, as it was the last line

## What did you see instead?
![20171103-1](https://user-images.githubusercontent.com/5621624/32378509-f013383a-c0d0-11e7-8c08-1e03c7ad44a9.png)

```
Exception of type 'ProtoCore.Exceptions.CompilerInternalException' was thrown.

   at ProtoCore.Utils.NodeUtils.Clone(AssociativeNode rhsNode)
   at System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at ProtoCore.AST.AssociativeAST.ExprListNode..ctor(ExprListNode rhs)
   at ProtoCore.Utils.NodeUtils.Clone(AssociativeNode rhsNode)
   at System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at ProtoCore.AST.AssociativeAST.FunctionCallNode..ctor(FunctionCallNode rhs)
   at ProtoCore.Utils.NodeUtils.Clone(AssociativeNode rhsNode)
   at ProtoCore.AST.AssociativeAST.IdentifierListNode..ctor(IdentifierListNode rhs)
   at ProtoCore.Utils.NodeUtils.Clone(AssociativeNode rhsNode)
   at ProtoCore.AST.AssociativeAST.BinaryExpressionNode..ctor(BinaryExpressionNode rhs)
   at ProtoCore.Utils.NodeUtils.Clone(AssociativeNode rhsNode)
   at ProtoScript.Runners.LiveRunner.CompileAndExecuteForDeltaExecution(List`1 astList)
   at ProtoScript.Runners.LiveRunner.SynchronizeInternal(GraphSyncData syncData)
   at ProtoScript.Runners.LiveRunner.UpdateGraph(GraphSyncData syncData)
   at Dynamo.Scheduler.UpdateGraphAsyncTask.HandleTaskExecutionCore()
   at Dynamo.Scheduler.AsyncTask.Execute()
```

**Comment by jnealb** [comment]
@vykrum Thanks. This is tracked with QNTM-1966.

**Comment by jnealb** [comment]
@vykrum This is fixed in the daily builds for 2.0.
